token_benchmark_ray.py

```diff
@@ -459,6 +467,15 @@ def run_token_benchmark(
         "name=foo,bar=1. These will be added to the metadata field of the results. "
     ),
 )
+args.add_argument(
+    "--log-prompts",
+    type=bool,
+    default=False,
```

Comment:
do you want the default to be false? 
we can only change this in the argo sensor

Reply:
Yes I think it's spammy by default

Reply:
So you don't want it to be spammy. alright